### DISCUSSION TITLE
Nationality and Citizenship

### DISCUSSION THREAD

**Discussion by ungeahnt** [FamilySearch/GEDCOM#193] [General]
Is there a distinction in Gedcom between **nationality** (NATI) and **citizenship** and how should a citizenship be represented in GEDCOM?

Naturalization (NATU) is not appropriate in many cases and on the other hand it describes an event (obtaining citizenship) and not the fact (one has citizenship).

Examples:
1) Seconded employee moves with his partner to the USA and a child is born there. After a few months/years, the entire family returns home. Then the child has the nationality and citizenship of the parents as well as US citizenship, but not US nationality.

2) Members of tribes / folks with their own nationality can have different citizenships.

**Comment by tychonievich** [comment]
You are correct that 7.0 does not have a citizenship attribute, only a citizenship-gaining event, and hence no way of documenting from-birth citizenship other than the `NATI` tag. An approved but not yet published change (#187) will expand the definition of NATI in 7.0.10; thus we recommend using

```gedcom
1 NATI United States
2 TYPE citizenship
```

There is some discussion of expanding the number of events and attributes in 7.1 (see #117) and a specific citizenship attribute may be added then.
